odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW

```diff
@@ -305,6 +316,14 @@ PRINT TITLE="BANNO.TEMPLATE.V1.POW"
       END
     END
   END
+
+ CALL BNOPRINTRGDATA
+ CALL BNOPRINTDEBUG
+ IF BNODEBUGMODE=BNODEBUGMODEOFF THEN
+  DO
+   NEWLINE
```

Comment:
Makes the JSON code a bit easier to read. Both the calls to PRINTBNORGDATA and BNOPRINTDEBUG add the JSON required comma at the start of the procedures so, if DEBUGMODE is off, the NEWLINE just bounces the 'cursor' down to the next line. Not "needed" like the BNO indent... just makes the JSON easier to read should it be necessary.

Reply:
Cool.  Yeah, I do understand the readability issue but find I usually like to run it through a formatter anyways since it is really hard, in the PO code, to keep indentation and newlines, etc. accurate.